### DISCUSSION TITLE
Error in API requests only shows login page

### DISCUSSION THREAD

**Discussion by ronnyandre** [firefly-iii/firefly-iii#4537] [Firefly III]
I am running Firefly III version 5.4.6.

**Description**
I have created a Python script that downloads my bank statement and formats the transactions ready to be inserted to Firefly. However, I have an error somewhere in my JSON data which I am sending to the Firefly API, but the API only returns the HTML for the login page and a HTTP status 200. No error message and no data gets inserted through the API. I am using personal token to talk to the API, if that has anything to do with the issue.

**Comment by JC5** [comment]
Make sure you add the header `Accept: application/json` so Firefly III knows you want a JSON based error.
